### PR TITLE
pacman, usermod, useradd, userdel, xclip, xbacklight, yay: add French translation

### PR DESCRIPTION
**Pull request by D34DPlayer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/linux/pacman.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+> Outil de gestion de paquets sur Arch Linux. 
+
+- Synchronise et mets à jour tous les paquets:
+
+`pacman -Syu`
+
+- Installe un nouveau paquet:
+
+`pacman -S {{package_name}}`
```
Comment (navarroaxel) [inline pages.fr/linux/pacman.md:11]: Please translate the arguments for get this easier for non-English speakers.

File: pages.fr/linux/pacman.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# pacman
+
+> Outil de gestion de paquets sur Arch Linux. 
+
+- Synchronise et mets à jour tous les paquets:
```
Comment (navarroaxel) [inline pages.fr/linux/pacman.md:5]: I remember that in French you should use a space before `:`:
```suggestion
- Synchronise et mets à jour tous les paquets :
```
Please update all the example descriptions.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @D34DPlayer, I left some comments.

File: pages.fr/linux/pacman.md (hunk 2 above)
Comment (D34DPlayer) [reply pages.fr/linux/pacman.md]: Yep, I added them.

File: pages.fr/linux/usermod.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# usermod
+
+> Modifie un compte utilisateur.
+
+- Change le nom d'un utilisateur:
```
Comment (navarroaxel) [inline pages.fr/linux/usermod.md:5]:
```suggestion
- Change le nom d'un utilisateur :
```

File: pages.fr/linux/usermod.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`usermod -l {{nouveau_nom}} {{utilisateur}}`
+
+- Ajoute l'utilisateur à des groupes supplementaires (attention à l'omission d'espaces):
```
Comment (navarroaxel) [inline pages.fr/linux/usermod.md:9]:
```suggestion
- Ajoute l'utilisateur à des groupes supplementaires (attention à l'omission d'espaces) :
```

File: pages.fr/linux/usermod.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+`usermod -a -G {{groupe1,groupe2}} {{utilisateur}}`
+
+- Crée un nouveau dossier home pour un utilisateur et déplace ses fichiers vers celui-ci:
```
Comment (navarroaxel) [inline pages.fr/linux/usermod.md:13]:
```suggestion
- Crée un nouveau dossier home pour un utilisateur et déplace ses fichiers vers celui-ci :
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just a few changes.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**